### PR TITLE
Delete obsolete local accounts on app startup

### PR DESCRIPTION
**Pull request by jkoenig134**
# Readiness checklist

- [x] I added/updated tests.
- [x] I ensured that the PR title is good enough for the changelog.
- [x] I labeled the PR.
- [x] I self-reviewed the PR.

# Description

Updates the runtime from 6.15.0 to 6.18.0

- https://github.com/nmshd/runtime/compare/6.17.0...6.18.0
- https://github.com/nmshd/runtime/compare/6.16.0...6.17.0
- https://github.com/nmshd/runtime/compare/6.15.0...6.16.0

I did not add `checkIfIdentityIsDeleted` from the `AccountFacade` b/c I don't want the app to use this.

### CODE REVIEW

**Comment by Milena-Czierlinski** [comment]
@jkoenig134 There are two occurrences of `deleteAccount` in `identity_deletion_processes_facade_test.dart` that should probably be changed to `offboard_account`.

File: packages/enmeshed_runtime_bridge/lib/src/services/facades/transport/identity_deletion_processes_facade.dart
Comment (Milena-Czierlinski) [inline packages/enmeshed_runtime_bridge/lib/src/services/facades/transport/identity_deletion_processes_facade.dart:1]: I thought we didn't want to expose the length of the grace period.

Comment (jkoenig134) [reply packages/enmeshed_runtime_bridge/lib/src/services/facades/transport/identity_deletion_processes_facade.dart:1]: I need this for manual end2end testing. See https://github.com/nmshd/runtime/pull/368 for even more details!

Comment (Milena-Czierlinski) [reply packages/enmeshed_runtime_bridge/lib/src/services/facades/transport/identity_deletion_processes_facade.dart:1]: It's okay, so we can use it for testing and demos. However, we never want to use it productively.

**Review by Milena-Czierlinski — APPROVED** [review]

**Pull request merged**